gems/activerecord/7.0/activerecord-7.0.rbs

```diff
@@ -4,4 +4,8 @@ module ActiveRecord
       def primary_abstract_class: () -> void
     end
   end
+
+  module Enum
+    def enum: (?(String | Symbol) name, ?(Hash[(Symbol | String), untyped] | Array[Symbol | String]) values, **untyped options) -> void
```

Comment:
This method definition is misleading. 
This method actually accepts Rails 6 style enum definitions like the following.

```ruby
enum status: { active: 1, inactive: 2 }
```

But the type definition does not have the corresponding type definition. It looks only accepts `enum :status, { active: 1, inactive: 2 }` style enum.


So I think we should add a branch to clarify it receives the Rails 6 style definition.


---

By the way, the test passes because `**untyped` matches `statsu: {...}` keyword. I think it is not unclear.

Reply:
Thank you for your review.
Exactly! So, I changed signatures to be able to use Rails 6 style enum definitions in Rails 7.